Example/ViewController.swift

```diff
@@ -682,3 +685,25 @@ extension ViewController: NavigationViewControllerDelegate {
         clearNavigationMapView()
     }
 }
+
+extension ViewController : StyleManagerDelegate {
```

Comment:
Let's move this to a different file and format it similarly to other extensions `ViewController+....swift`

Reply:
Thanks for the suggestion, I'll move it to another file.